modules/md5sum/main.nf

```diff
@@ -0,0 +1,35 @@
+process MD5SUM {
+    tag "$meta.id"
+    label 'process_low'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using bcl-convert. Please use docker or singularity containers."
```

Comment:
This looks like it is leftover from your previous module

Reply:
Whoops, that damned copy paste again!